web/frontend/.eslintrc.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
     'max-params': 'off',
     'max-statements': 'off',
     'multiline-ternary': ['error', 'always-multiline'],
+    "no-shadow": "off",
```

Comment:
@micheal-parks I need your advice here. I got the following error.

```rdk/web/frontend/src/components/base.vue
  19:12  error  'Keymap' is already declared in the upper scope on line 19 column 12  no-shadow```

Reply:
There's a fix for this that was implemented [in the typescript SDK](https://github.com/viamrobotics/viam-typescript-sdk/blob/main/.eslintrc.js) (see lines 61 & 74), we should do the same here

Reply:
Yea I already add `"no-shadow": "off"` and this did fix the linting error.

Reply:
we need to also add what is on line 74 to keep the rule (which is a good rule)

Reply:
Okay adding!